src/main/pages/che-6/user-guide/version-control.adoc

```diff
@@ -9,55 +9,85 @@ folder: che-6/user-guide
 
-The following sections describe how to connect and authenticate users to a remote git repository. Any operations that involve authentication on the remote repository will need to be done via the IDE interface unless authentication is setup separately for the workspace machine(terminal/commands).
+The following sections describe how to connect and authenticate users to a remote Git repository. Any operations that involve authentication on the remote repository must be done via the IDE interface unless authentication is set up separately for the workspace machine (through the terminal or commands).
 
-Private repositories will require a secure SSH connection and most developers who plan to push to a repo will clone it via SSH, so an SSH key pair needs to be generated. SSH keys are saved in user preferences, so you need to generate the SSH key only once and it will be used in all workspaces.
+An SSH key pair must be generated because private repositories require a secure SSH connection and most developers who plan to push to a repository clone the repository via SSH. SSH keys are saved in the user preferences, so you need to generate the SSH key only once and it will be used in all workspaces.
 
-*Note: HTTPS git URL can only be used to push changes if you enable oAuth authentication as described in link:user-management.html#social-login-and-brokering[GitHub oAuth]*.
+[NOTE]
+====
+HTTPS Git URL can only be used to push changes if you enable oAuth authentication as described in link:user-management.html#social-login-and-brokering[GitHub oAuth].
+====
 
-[id="generateupload-ssh-keys"]
-== Generate/Upload SSH Keys
+[id="generating-and-uploading-ssh-keys"]
+== Generating and uploading SSH keys
 
-SSH keys can be generated at `Profile > Preferences > SSH > VCS`. Use the `Generate Key` button and manually save the resulting key to your Git hosting provider account. When prompted to provide the hostname for your repo, make sure it is a bare hostname (no www or http/https) as in the example below. After the key has been generated, you can view and copy it, and save to your repository hosting account.
+To generate the SSH keys, take the following steps:
 
-*IMPORTANT!*. Hostname is an actual hostname of a VCS provider. Examples: `github.com`, `bitbucket.org`.
+. Click *Profile* > *Preferences* > *SSH* > *VCS*.
 
-*Use Existing SSH Keys*
+. Click *Generate Key* and manually save the resulting key to your Git hosting provider account. 
 
-You can upload an existing public key instead of creating a new SSH key. When uploading a key add the hostname (using no *www or http/https* - as in the example below). Note that the `public key > view` button will not be available with this option as the public file should be generated already.
+. When prompted to provide the hostname for your repository, ensure that it is a bare hostname (without www or http/https). Hostname is an actual hostname of a VCS provider. Example: `github.com`, `bitbucket.org`
```

Comment:
I'd mark up "`www` or `http`/`https`" somehow. You've used monospace for `github.com` and `bitbucket.org` here, so I think that would make sense. :)